F21/megannigg/lowercase.py

```diff
@@ -0,0 +1,30 @@
+def lowercase(word, language):
+
+#Turkish or Azerbaijani uses lowercase i with out a dot    
+    if 'tr' in language or 'az' in language:
+        word = word.replace('I','ı')
+        word = word.lower()
+        
+#Irish rule: when converting uppercase vowels to lowercase, the vowel must be proceded by a dash
+    elif 'ga' in language:
+        if len(word)>1:
+            if word[0] in 'tn' and word[1] in 'AEIOUÁÉÍÓÚ':
```

Comment:
This will only work as expected on the test cases if you do some Unicode normalization first.